core/src/main/scala/org/genivi/sota/core/AutoInstallResource.scala

```diff
@@ -89,7 +61,7 @@ class AutoInstallResource
             removeAll(ns, pkgName)
           }
         } ~
-        devicePathExtractor(ns) { devUuid =>
+        devicePathExtractor(ns)(ec) { devUuid =>
```

Comment:
does this execution context have to be passed explicitly? If not I think leaving it out is better.